pct.sh

```diff
@@ -40,7 +40,8 @@ MAVEN_PROPERTIES+=":jenkins.version=${JENKINS_VERSION}:overrideWar=$(pwd)/megawa
 # TODO When all plugins in the managed set are using a plugin parent POM with HPI Plugin 3.29 or
 # later (i.e., plugin parent POM 4.44 or later), this can be deleted.
 #
-MAVEN_PROPERTIES+=:hpi-plugin.version=3.30
+# TODO https://github.com/jenkinsci/maven-hpi-plugin/pull/368
+MAVEN_PROPERTIES+=:hpi-plugin.version=3.31-rc1289.8ccff91f4e1a
```

Comment:
huh pretty useful being able to force HPI higher, not sure this is even a hack at this point. Especially when we need to fix a potential bug.